Jenkinsfile

```diff
@@ -64,15 +64,16 @@ node ('master') {
         // Use a docker container to build and protogen, so that the Jenkins
         // environment doesn't need all the dependencies.
         stage("Build Test Dependencies") {
-            sh './bin/build_all'
+            sh 'docker-compose build'
         }
 
         stage("Run Lint") {
-            sh 'docker run --rm -v $(pwd):/project/sawtooth-core sawtooth-dev-go:$ISOLATION_ID run_go_fmt'
+            sh 'docker run --rm sawtooth-seth-cli:latest run_go_fmt'
```

Comment:
Using latest here is not valid. If there are two PRs running in parallel, `latest` may not point to the image you just built (it is a race condition). That is why ISOLATION_ID is used in the other one.